### PR TITLE
feat: product listing page

### PR DESCRIPTION
**Pull request by shashidwivedi**
- Add product listing page
- Add filters - by pricing, by category, by ratings

127 0 0 1_5500_pages_product-listing-page html

### CODE REVIEW

**Review by Sameony — APPROVED** [review]
beautiful minimal design of the website with utility classes from component library, love it!
nice and clean code.

File: pages/product-listing-page.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        </div>
+      </div>
+    </main>
+    <footer class="footer">
+      <p class="text-center text-gray text-small">
+        <span>© Pillow Willow. All rights reserved.</span>
+      </p>
+    </footer>
+  </body>
+</html>
```
Comment (vaibhav-mahalle) [inline pages/product-listing-page.html:219]: dont commit empty files in your PR.

**Review by vaibhav-mahalle — APPROVED** [review]
everything else looks good. suggested some changes.

File: pages/product-listing-page.html (hunk 1 above)
Comment (shashidwivedi) [reply pages/product-listing-page.html:219]: Yes, Vaibhav. Will absolutely take care of this next time. Could have committed empty files on a separate branch named setup.

**Review by shivangisharma01 — COMMENTED** [review]
just one suggestion do not add empty file's before hand  because sometimes it create problem other than that its good.

**Review by shivangisharma01 — COMMENTED** [review]
Good use of variable, the code is clean.

**Pull request merged**